src/content/app/genome-browser/components/track-settings-panel/TrackSettingsPanel.tsx

```diff
@@ -24,6 +26,7 @@ import { TrackType } from 'src/content/app/genome-browser/state/track-settings/t
 
 type Props = {
   trackId: string;
+  onClose: () => void;
```

Comment:
I suggest we rename this property to `onOutsideClick`. If you think from `TrackSettingsPanel`'s standpoint, it does not know anything about opening or closing. The only thing it knows about is that someone clicked outside of it.